go/ql/test/library-tests/semmle/go/dataflow/FlowSteps/LocalTaintStep.expected

```diff
@@ -3,13 +3,6 @@
 | main.go:38:13:38:13 | 1 | main.go:38:7:38:20 | slice literal |
 | main.go:38:16:38:16 | 2 | main.go:38:7:38:20 | slice literal |
 | main.go:38:19:38:19 | 3 | main.go:38:7:38:20 | slice literal |
-| main.go:39:15:39:15 | s | main.go:39:8:39:25 | call to append |
```

Comment:
You're sure taint still works as expected here, just via different steps?

Reply:
This test, and the value flow counterpart, only show steps without any reads or stores. The model for append is now a value step with a store, so it isn't shown. The test in `go/ql/test/library-tests/semmle/go/dataflow/ExternalTaintFlow/test.go` still shows taint flow through `append`.